common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -20,7 +20,7 @@ ArduPilot has support for two notch filters whose filter frequency can be linked
 
 While the lowpass filters can effectively diminish the impact of this noise, having low frequency set points creates a lot of phase lag and therefore reduces how aggressive the tune can be before oscillation occurs, which results in a poorer tune.
 
-For the gyro based rate controllers, this reduces their ability to respond to fast disturbances. If the gyro lowpass filter can be set higher, the phase lag induced is lower and the tune can be more aggressive. But this allows more noise and vibration, effectively canceling that gain out. Enabling either one or both of the Harmonic Notch filters allow targeting the noise generated by the motors, allowing a higher frequency of the following low pass to be set and therefore allowing tighter tune.
+For the gyro based rate controllers, this reduces their ability to respond to fast disturbances. If the gyro lowpass filter is set higher, the phase lag induced is lower and the tune can be more aggressive. But this allows more noise and vibration, effectively canceling that gain out. Enabling either one or both of the Harmonic Notch filters gives the ability to target the noise generated by the motors, allowing a higher frequency of the following low pass to be set and therefore allowing tighter tune.
```

Comment:
I disagree with the first change.....raising the lowpass filter may NOT be possible if the noise cannot be effectively filtered....second change is okay

Reply:
This phrase is a bit long; a bit hard to wrap my head around it.

With the original wording it reads to me:
1. If the knee frequency can be set higher (depends on what?)
2. then the filter phase lag is lower (agreed)
3. then the tune has more bandwidth to work with (agreed) and can be tuned more aggressively.
4. But then the more aggressive control input will bring on more noise and vibration, cancelling the merits (agreed).

What I tried to do was do away with the unmentioned dependency of #1 to simplify the phrase.

But no worries, I'll adapt as you ask, it's no biggie.